### PR TITLE
chore(ci): remove unused e2e test

### PR DESCRIPTION
**Pull request by BowenXiao1999**
## What's changed and what's your intention?

Remove batch/streaming e2e test first.

Next step will rename the v2 tests into first class citizen

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests

### CODE REVIEW

File: .github/workflow-template/jobs/e2e-source.yml [hunk 1]
```diff
@@ -94,11 +94,6 @@ jobs:
           chmod +x ./target/debug/risedev-playground
           chmod +x ./target/debug/risingwave_regress_test
 
-      - name: Run batch tests on Postgresql
-        timeout-minutes: 10
-        run: |
-          sqllogictest -p ${{ env.PG_PORT }} './e2e_test/batch/**/*.slt'
-
```
Comment (BowenXiao1999) [inline .github/workflow-template/jobs/e2e-source.yml:101]: Do not know why e2e-source has a test on PG. Looks unnecessary so delete. cc @shanicky @tabVersion

Comment (skyzh) [reply .github/workflow-template/jobs/e2e-source.yml:101]: This is used by regression test?

Comment (skyzh) [reply .github/workflow-template/jobs/e2e-source.yml:101]: cc @liurenjie1024

**Review by skyzh — APPROVED** [review]
May also help updating CONTRIBUTING.md

File: .github/workflow-template/jobs/e2e-source.yml (hunk 1 above)
Comment (neverchanje) [reply .github/workflow-template/jobs/e2e-source.yml:101]: It's to make ensure that we have the same query result as pg, I think it's necessary.

Comment (BowenXiao1999) [reply .github/workflow-template/jobs/e2e-source.yml:101]: No. If regression tests, at least RW should also run it. However RW is not.

File: e2e_test/streaming/nexmark/create_views.slt.part [hunk 2]
```diff
@@ -1,4 +0,0 @@
-include ./views/q0.slt.part
```
Comment (neverchanje) [inline e2e_test/streaming/nexmark/create_views.slt.part:1]: There are no nexmark tests in v2/ so pls do not remove these files

File: e2e_test/streaming/tpch/q1.slt.part [hunk 3]
```diff
@@ -1,7 +0,0 @@
-query TTRRRRRRRI
```
Comment (neverchanje) [inline e2e_test/streaming/tpch/q1.slt.part:1]: tpch have no copy in v2/ as well. pls keep them.

File: .github/workflow-template/jobs/e2e-source.yml (hunk 1 above)
Comment (liurenjie1024) [reply .github/workflow-template/jobs/e2e-source.yml:101]: No, it's not used by regression test.

**Review by liurenjie1024 — APPROVED** [review]
LGTM

**Review by TennyZhuang — APPROVED** [review]
LGTM

**Comment by huangjw806** [comment]
LGTM

**Pull request merged**